### PR TITLE
Some player stuff

### PR DESCRIPTION
**Pull request by sargunv**

### CODE REVIEW

File: mappings/net/minecraft/entity/LivingEntity.mapping [hunk 1]
```diff
@@ -112,6 +114,7 @@ CLASS aii net/minecraft/entity/LivingEntity
 	METHOD cx isUndead ()Z
 	METHOD cy getHealth ()F
 	METHOD cz getRecentDamageSource ()Lahi;
+	METHOD d dropAllDrops (Lahi;)V
```
Comment (liach) [inline mappings/net/minecraft/entity/LivingEntity.mapping]: Is this dropping with respect to damage source? Needs a better name.

Comment (sargunv) [reply mappings/net/minecraft/entity/LivingEntity.mapping]: IMO the name makes sense in the context of the other methods around it. It's near `dropInventory`, `dropEquipment`, and `dropLoot`, and it calls all three of them, so `dropAllDrops`.

Comment (liach) [reply mappings/net/minecraft/entity/LivingEntity.mapping]: Then you should just name it `drop`. You need to be concise.

Comment (sargunv) [reply mappings/net/minecraft/entity/LivingEntity.mapping]: sure

**Review by liach — APPROVED** [review]

**Review by Gegy — APPROVED** [review]

**Pull request merged**